go.mod

```diff
@@ -26,9 +27,9 @@ require (
 	github.com/urfave/cli v1.22.9
 	go.etcd.io/bbolt v1.3.7
 	go.uber.org/mock v0.4.0
-	golang.org/x/sync v0.6.0
-	google.golang.org/grpc v1.59.0
-	google.golang.org/protobuf v1.33.0
+	golang.org/x/sync v0.8.0
+	google.golang.org/grpc v1.64.1
```

Comment:
Same question here re the version we choose for `google.golang.org/grpc` and `google.golang.org/protobuf`. IMO they should be the same for all projects, as we've had issues with mismatches and updates to new versions before...

Reply:
Given that `frdrpc` depends on GRPC v1.65.0 and protobuf v1.34.2 I bumped to those versions. Also added a new forked hex display (see here: https://github.com/lightninglabs/protobuf-go-hex-display/pull/3). PTAL